src/pxl_scripts.tsx

```diff
@@ -22,19 +22,24 @@ const customQuery = require('pxl_scripts/custom-query.json');
 const exampleQuery = require('pxl_scripts/example-query.json');
 const httpDataFiltered = require('pxl_scripts/http-data-filtered.json');
 const httpErrorsPerService = require('pxl_scripts/http-errors-per-service.json');
+const podMetrics = require('pxl_scripts/pods-metrics.json');
 
-interface Script {
+export interface Script {
   name: string;
   description: string;
   script: string;
+  isTabular: boolean;
+  columnNames: string[];
```

Comment:
Since these may be not present in some scripts, you can make these fields optional.
```suggestion
  isTabular?: boolean;
  columnNames?: string[];
```